configure.ac

```diff
@@ -1094,6 +1094,12 @@ AS_IF([test x$enable_coverage = xyes],
       ]
 )
 
+AX_CHECK_COMPILE_FLAG([-Wno-array-bounds],
+  [M0_CFLAGS="-Wno-array-bounds $M0_CFLAGS"])
```

Comment:
Maybe it would be better to not disable this warning and fix the issue it warns about like this:

```diff
--- a/xcode/xcode.c
+++ b/xcode/xcode.c
@@ -102,7 +102,7 @@ bool m0_xcode_type_invariant(const struct m0_xcode_type *xt)
                _0C(xt->xct_nr >= min[xt->xct_aggr]) &&
                _0C(xt->xct_nr <= max[xt->xct_aggr]) &&
                m0_forall(i, xt->xct_nr, ({
-                       const struct m0_xcode_field *f = &xt->xct_child[i];
+                       const struct m0_xcode_field *f = xt->xct_child + i;
 
                        field_invariant(xt, f) &&
                        /* field doesn't overlap with the previous one */
```